_author_instructions/01_policies.md

```diff
@@ -333,6 +337,10 @@ Authors are responsible for obtaining all needed permissions to include previous
 If an article is an adaptation of a previously published article, it must be noted in the submission cover letter and major changes noted.
 Evaluating whether such changes constitute a significant revision will be part of the review process.
 
+The U.S. National Science Foundation definition of plagiarism as "the appropriation of another person’s ideas, processes, results, or words without giving appropriate credit" (45 Code of Federal Regulations, Section 689.1). Authors should not engage in plagiarism, which operationally means verbatim or near-verbatim copying, or even very close paraphrasing, of text or results from another’s work. 
```

Comment:
"of plagiarism as" -> "of plagiarism is", or could be "defines plagiarism as..."